joomla/joomla-cms
# Add stylesheet version to all css files and replace deprecated stylesheet signatures usage

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/9638 (css part).

### Summary of Changes

This does for css the same that was done in https://github.com/joomla/joomla-cms/pull/12373 for js.

So, this PR replaces all deprecated stylesheets methods signature usage across the CMS.

Also:

- adds stylesheet version to all js files to solve https://github.com/joomla/joomla-cms/pull/9638
- use the api in conditional ie statements

### Testing Instructions

1. Code review.
2. Use latest staging
3. Apply https://github.com/joomla/joomla-cms/pull/12375 and them this patch
4. Navigate in the frontend/backend/installation (several template). Confirm all is working fine and check all external css files are loaded and with media version

### Documentation Changes Required

None.

### Notes

Unit tests will fail until https://github.com/joomla/joomla-cms/pull/12375 is merged.
